opensrp-chw/src/main/java/org/smartregister/chw/common/Helper.java

```diff
@@ -0,0 +1,21 @@
+package org.smartregister.chw.common;
```

Comment:
lets make this class name more descriptive,or may be having a method on the calling class ?

Reply:
Any ideas?

Reply:
If this is the only function that would be implemented on the helper, just implement it on the calling class.

Reply:
okay